tests/TestCase/I18n/Parser/PoFileParserTest.php

```diff
@@ -65,39 +65,39 @@ public function testParse()
         $expected = [
             'Plural Rule 1' => [
                 '_context' => [
-                    '' => 'Plural Rule 1 (translated)',
-                ],
```

Comment:
You shouldnt revert the trailing commas IMO, just too much diff changes then that are none.

Reply:
Seriously? All the other lines haven't had one I think and AFAIK best practice by CS is to not have it there?

Reply:
No best practice (what the sniffer should be enforcing) is to always have the multiline trailing commas, to keep diffs low exactly for the reasons shown in this PR: Too much unrelated noise around the few lines of actual change.

Reply:
They aren't used everywhere either here! https://github.com/cakephp/cakephp/blob/master/tests/TestCase/I18n/Parser/PoFileParserTest.php#L59-L102 So either remove them all or add them all!? However both ways will cause "noise".

Reply:
I guess in that case you can ignore my comment.

Reply:
We should agree on something and putt it into the coding standard? What do you think?

Reply:
Yeah, but can then be unrelated to this PR.